### ISSUE TITLE
Boss fight not resetting

### ISSUE THREAD

**Issue by MoireMoire** [NRG-Wow/NRGWow-Bugtracker#117]
[CATA][DUNGEONS]
**Reporter(Player/Character) Name**: Moire

**Description**: Boss fight not resetting after a wipe

**Steps to reproduce the problem**: Happened with Erudax, Forgemaster Throngus and a couple other bosses. If we wipe and the boss is far (like 60 yards or more) from where he usually stands/got pulled, might happen that he gets stuck or bugs in some way not resetting the bossfight, resulting in the group not being able to enter in the dungeon to complete it.

1. Enter a dungeon, most likely Grim Batol
2.  Engage a boss 
3. Kite it and die

**Comment by Illusie** [comment]
Yep you right and there is another thing about Dungeons I gonna open a report now.
+1
